google/cloud/speech/v1beta1/cloud_speech_gapic.yaml

```diff
@@ -0,0 +1,156 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
+language_settings:
+  java:
+    package_name: com.google.cloud.speech.spi.v1beta1
+  python:
+    package_name: google.cloud.speech.v1beta1
+  go:
+    package_name: cloud.google.com/go/speech/apiv1beta1
+  csharp:
+    package_name: Google.Cloud.Speech.V1beta1
+  ruby:
+    package_name: Google::Cloud::Speech::V1beta1
+  php:
+    package_name: Google\Cloud\Speech\V1beta1
+  nodejs:
+    package_name: "@google-cloud/speech"
+interfaces:
+- name: google.cloud.speech.v1beta1.Speech
+  collections: []
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 60000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 60000
+    total_timeout_millis: 600000
+  methods:
+  - name: SyncRecognize
+    flattening:
+      groups:
+      - parameters:
+        - config
+        - audio
+    required_fields:
+    - config
+    - audio
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: AsyncRecognize
+    flattening:
+      groups:
+      - parameters:
+        - config
+        - audio
+    required_fields:
+    - config
+    - audio
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: StreamingRecognize
+    flattening:
+      groups:
+      - parameters:
+        - streaming_config
+        - audio_content
+    required_fields:
+    - streaming_config
+    - audio_content
```

Comment:
Remove the flattening and required_fields configs, we will not use them for bidi-streams probably.